.github/workflows/test.yml

```diff
@@ -0,0 +1,64 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Test
+
+on: 
+  push:
+    branches: 
+    - main
+  pull_request:
+    branches: 
+    - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.9]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        env:
+          GITHUB_TOKEN: ${{ secrets.NUMBERS_GITHUB_TOKEN }}
+        run: |
+          sudo apt-get update && DEBIAN_FRONTEND=noninteractive sudo apt-get install -y swig libexiv2-dev libboost-python-dev
+          python -m pip install --upgrade pip
+          pip install pytest
```

Comment:
```suggestion
          python3 -m pip install --upgrade pip
          python3 -m pip install pytest
```

It's always a good practice to use `python3` instead of `python` when invoking python3 intepreter.